### ISSUE TITLE
Finetune Error

### ISSUE THREAD

**Issue by 0xuye0** [Tencent/tencent-ml-images#35]
finetune报错如下
![image](https://user-images.githubusercontent.com/5190912/49695655-3aeebe80-fbd9-11e8-9439-268daa9ef309.png)
尝试了好久没有解决。。不知道会是何原因呢。。

我的环境是python3+tensorflow1.10.1

**Comment by 0xuye0** [comment]
补充一下，是在finetune.py的
 _, step, loss = sess.run([train_op, global_step, Loss])
这一行报错

**Comment by AIML** [comment]
看你错误的位置是你的tfrecord解析脚本中feature的定义有问题；请查看示例代码中tfrecord的生成脚本。label是一个one_hot类型的vector，被编码成了一个string，要使用decode_raw解码出来

**Comment by scriptboy1990** [comment]
> finetune报错如下
> ![image](https://user-images.githubusercontent.com/5190912/49695655-3aeebe80-fbd9-11e8-9439-268daa9ef309.png)
> 尝试了好久没有解决。。不知道会是何原因呢。。
> 
> 我的环境是python3+tensorflow1.10.1

请问解决了这个问题了吗。

**Comment by cherry0619** [comment]
> 看你错误的位置是你的tfrecord解析脚本中feature的定义有问题；请查看示例代码中tfrecord的生成脚本。label是一个one_hot类型的vector，被编码成了一个string，要使用decode_raw解码出来

it  works！！
